packages/cursorless-engine/src/actions/Sort.ts

```diff
@@ -11,6 +11,10 @@ abstract class SortBase implements Action {
   protected abstract sortTexts(texts: string[]): string[];
 
   async run([targets]: [Target[]]): Promise<ActionReturnValue> {
+    if (targets.length < 2) {
+      throw new Error("Require more than one targets with this action");
```

Comment:
```suggestion
      throw new Error('This action requires than one target. For example, instead of "sort block", you might say "sort every line block" to sort the lines in a block.  Or you might say "reverse every key" to reverse the order of the keys in a map.');
```

@josharian is that the kind of error message you had in mind?

Reply:
I don't think we should put verbose error messages on just this action. A lot of our different actions already have error messages for to few targets and they are as terse as the above one. I would argue for uniformity.

Reply:
Maybe link to page in docs that elaborates then? Just because we have terse messages elsewhere doesn't make it a good idea 😄

Reply:
I'm not against verbose error messages, but I am very much against treating some features like they are more important than others. I most definitely think that we should have a uniform error message system. We probably shouldn't use just the base error either and have more custom and verbose messages overall. Linking to the docs is not bad idea.

Reply:
I also wonder if we want this to be a warning rather than an error.  I could see for example something like "scout air" then "sort every key scout next", and then popping to repeat a few times, to sort the keys in a bunch of maps which have a particular token in them.  I know it's a bit far fetched, but in that case, I might end up sorting a map with one element along the way, and I don't want the command chain to break.  Technically it is completely valid to sort a single target, it just so happens to be a no-op.  I'm happy to ignore the warning in that case.  Tbh it would be nice to be able to just check a box to suppress it in the future, but that's extra credit.

Reply:
I agree with that.

Reply:
My 2c, generally:

Warning > error

The best messages are both concise and helpful. But if I had to pick one, I'd choose helpful.

Maybe something like:

> ACTION works on multiple targets, e.g. "sort every line block" instead of "sort block".

Where ACTION is the actual action used, if we have ready access to it.

Reply:
I think we need to discuss this

Reply:
done